src/client/sandbox/code-instrumentation/location/index.js

```diff
@@ -34,11 +37,17 @@ export default class LocationAccessorsInstrumentation extends SandboxBase {
 
         window[INSTRUCTION.setLocation] = (location, value) => {
             if (isLocation(location)) {
-                location = value;
+                var resourceType = null;
 
-                return location;
-            }
+                if (window.self !== window.top) {
+                    value        = destLocation.resolveUrl(value, window.top.document);
+                    resourceType = stringifyResourceType(true);
```

Comment:
Can we have full form here: `stringifyResourceType(true, false, false)`? Also, it took me a while to figure out what it does. First of all `stringify` suggest that you pass something to be stringified. It more likely `getResourceTypeString`. The second issue it's not clear what arguments mean here. In such cases it's always better to have options object:
`getResourceTypeString({ isIframe: true})`